client/src/styles/_variables.scss

```diff
@@ -2,7 +2,13 @@ $color-red: #ff0000;
 $color-grey-black: #525656;
 $color-grey: #dcdcdc;
 $color-grey-dark: #aaaaaa;
+$empty-photo: #eeeeee;
 $color-snow: #f4f4f4;
 $color-black: #000000;
 $dark-red: #c40e0e;
+$color-button: #dc3545;
+$color1: #bdbdbd;
+$color2: #fafafa;
+$color3: #eeeeee;
```

Comment:
u can take a name for color from here
http://chir.ag/projects/name-that-color/#001489

Reply:
Thanks.I used the exact names from this site.